static/default.css

```diff
@@ -656,11 +656,22 @@ kbd {
 	display: block;
 	padding: 0 .5rem;
 }
+.auth-links__admin-box {
+    flex-shrink: 0
```

Comment:
Nitpick: spaces instead of tabs (the latter is preferred) + no semicolon. @bouncepaw, an automatic linter when?